packages/fork-choice/test/unit/protoArray/computeDeltas.test.ts

```diff
@@ -303,12 +303,9 @@ describe("computeDeltas", () => {
     // 1st validator is part of an attester slashing
     const equivocatingIndices = new Set([0]);
     let deltas = computeDeltas(indices.size, votes, balances, balances, equivocatingIndices);
-    expect(deltas[0]).to.be.equals(
-      -1 * (firstBalance + secondBalance),
-      "should disregard the 1st validator due to attester slashing"
-    );
-    expect(deltas[1]).to.be.equals(secondBalance, "should move 2nd balance from 1st root to 2nd root");
```

Comment:
the assertions are not obvious in my opinion, would prefer to keep those as comments (or use toBeWithMessage)